vendor.conf

```diff
@@ -113,10 +113,13 @@ github.com/bsphere/le_go                            7a984a84b5492ae539b79b62fb4a
 
 # gcplogs deps
 golang.org/x/oauth2                                 bf48bf16ab8d622ce64ec6ce98d2c98f916b6303
-google.golang.org/api                               de943baf05a022a8f921b544b7827bacaba1aed5
-go.opencensus.io                                    c3ed530f775d85e577ca652cb052a52c078aad26 # v0.11.0
-cloud.google.com/go                                 0fd7230b2a7505833d5f69b75cbd6c9582401479 # v0.23.0
-github.com/googleapis/gax-go                        317e0006254c44a0ac427cc52a0e083ff0b9622f # v2.0.0
+google.golang.org/api                               02490b97dff7cfde1995bd77de808fd27053bc87 # v0.7.0
+github.com/golang/groupcache                        869f871628b6baa9cfbc11732cdf6546b17c1298
```

Comment:
This replaced the dependency on `github.com/hashicorp/golang-lru`, which was Mozilla licensed and this one is Apache 2